.github/workflows/c-tests.yml

```diff
@@ -43,6 +43,8 @@ jobs:
       - name: Install dependencies OS X
         run: |
           brew install coreutils
+          brew install openssl
+          brew link openssl --force
```

Comment:
Only adding export OPENSSL_ROOT_DIR="/usr/local/opt/openssl" does not work, because it seems that every run opens a new bash.